### PR TITLE
223 clean git repo

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [ ] Your PR is linked to an issue.
- [ ] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [ ] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [ ] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue #223.

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 6bf1691c69778d8863117502de316ed3a1fc9d77: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

File: README.md [hunk 1]
```diff
@@ -17,10 +17,10 @@ The MASVS is a sister project of the [OWASP Mobile Security Testing Guide](https
 
 # Getting the MASVS
 
-PDF/Mobi/Epub/Docx downloads are available on the [Releases page](https://github.com/OWASP/owasp-masvs/releases "Releases"). The current release is [MASVS version 1.1](https://github.com/OWASP/owasp-masvs/releases/download/1.1/OWASP_Mobile_AppSec_Verification_Standard_v1.1.pdf). The MASVS is also available in different languages:
+PDF/Mobi/Epub/Docx downloads are available on the [Releases page](https://github.com/OWASP/owasp-masvs/releases "Releases"). The current release is [MASVS version 1.1.3](https://github.com/OWASP/owasp-masvs/releases/tag/1.1.3). The MASVS is also available in different languages:
```
Comment (sushi2k) [inline README.md]: description in link is missing:
-> 1.1.3 "MASVS version 1.1.3")

Comment (sushi2k) [reply README.md]: is [MASVS version 1.1.3] -> of the [MASVS is version 1.1.3]

File: README.md [hunk 2]
```diff
@@ -17,10 +17,10 @@ The MASVS is a sister project of the [OWASP Mobile Security Testing Guide](https
...
 
-- [Spanish](https://github.com/OWASP/owasp-masvs/releases/download/1.0-ES/OWASP_Mobile_AppSec_Verification_Standard_v1.0-ES.pdf)  
-- [Russian](https://github.com/OWASP/owasp-masvs/releases/download/1.1-RU/OWASP_Mobile_AppSec_Verification_Standard_v1.1-RU.pdf)
+- [Spanish](https://github.com/OWASP/owasp-masvs/tree/master/Document-es)  
+- [Russian](https://github.com/OWASP/owasp-masvs/tree/master/Document-ru)
 - [German](https://github.com/OWASP/owasp-masvs/tree/master/Document-de)
 - [French](https://github.com/OWASP/owasp-masvs/tree/master/Document-fr)
 - [Japanese](https://github.com/OWASP/owasp-masvs/tree/master/Document-ja)
```
Comment (sushi2k) [inline README.md]: Should have the language in quotes at the end, like:
Document-es "Spanish")

File: README.md [hunk 3]
```diff
@@ -32,6 +32,7 @@ Read it on [Gitbook](https://mobile-security.gitbook.io/masvs/ "GitBook Mobile A
 
 ## Create new PDF, Epub or Mobi
 
+You can find the documents in the release-page, if you want to generate the documents yourself, try to do the following.
```
Comment (sushi2k) [inline README.md]: release-page, if -> release page. If
try to do the following. -> execute the following steps.

**Comment by OMTGreleaser** [comment]
Results for commit 2d7d0a7eccd89af7230f02d6b5fddb787f9b8caf: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Comment by OMTGreleaser** [comment]
Results for commit c8342dc240fd632e70f92c79b7c7d0f70ccfb527: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Pull request merged**